rubocop.yml

```diff
@@ -64,6 +64,21 @@ Layout/AccessModifierIndentation:
 Layout/MultilineMethodCallIndentation:
   EnforcedStyle: indented
 
+Layout/MultilineArrayLineBreaks:
+  Enabled: true
+
+Layout/MultilineHashKeyLineBreaks:
+  Enabled: true
+
+Layout/MultilineMethodArgumentLineBreaks:
+  Enabled: true
+
+Layout/MultilineMethodParameterLineBreaks:
```

Comment:
Do you need this since you ignore method definitions below? Or is this in case you use line breaks, but it's not required to do so?

Reply:
Yes, it should not be required to do line breaks, but if you want to it should be one argument per line.